README.md

```diff
@@ -41,28 +41,40 @@ At this time pya is more suitable for offline rendering than realtime.
 ## Authors and Contributors
 
 * Thomas Hermann, Ambient Intelligence Group, Faculty of Technology, Bielefeld University (author and maintainer)
```

Comment:
suggestion:

Thomas (Author)
Jiajun (co-author and maintainer)
Alex (maintainer)

Reply:
Furthermore, 
Alexander Neumann, CareTech OWL, Bielefeld University of Applied Sciences

Reply:
oops, you should have been listed here for a loooong time! Thanks for updating!

Reply:
oh, I am :). I was suggesting to adjust the roles as the maintenance is primarily done by Jiajun (with my assistance) these days. And since Jiajun is already updating affiliations I'd request an update as well. So Thomas (author), Jiajun (co-author and maintainer (or MAINtainer ;))) and Alex (co-maintainer) maybe fits the roles the best. Not that this matters much.